src/data/state_incentives.ts

```diff
@@ -18,77 +21,142 @@ export type LowIncomeAuthority =
+  short_description: LocalizableString;
+  program_status: string;
+  program_start_raw?: string;
+  program_end_raw?: string;
+  payment_methods: PaymentMethod[];
+  rebate_value: string;
+  amount: Amount;
+  bonus_description?: string;
+  equipment_standards_restrictions?: string;
+  equipment_capacity_restrictions?: string;
+  contractor_restrictions?: string;
+  income_restrictions?: string;
+  filing_status?: FilingStatus;
+  owner_status: OwnerStatus[];
+  other_restrictions?: string;
+  stacking_details?: string;
+  financing_details?: string;
+  editorial_notes?: string;
+  questions?: string;
+  serve_in_api?: boolean;
+};
+
+const collectedIncentivePropertySchema = {
+  id: { type: 'string' },
+  data_urls: { type: 'string' },
+  authority_type: { type: 'string', enum: Object.values(AuthorityType) },
+  authority_name: { type: 'string' },
+  program_title: { type: 'string' },
+  program_url: { type: 'string' },
+  item: { type: 'string', enum: ALL_ITEMS },
+  item_if_selected_other: { type: 'string', nullable: true },
+  short_description: { $ref: 'LocalizableString' },
+  program_status: { type: 'string' },
+  program_start_raw: { type: 'string', nullable: true },
+  program_end_raw: { type: 'string', nullable: true },
+  payment_methods: {
+    type: 'array',
+    items: { type: 'string', enum: Object.values(PaymentMethod) },
+  },
+  rebate_value: { type: 'string' },
+  amount: AMOUNT_SCHEMA,
+  bonus_description: { type: 'string', nullable: true },
+  equipment_standards_restrictions: { type: 'string', nullable: true },
+  equipment_capacity_restrictions: { type: 'string', nullable: true },
+  contractor_restrictions: { type: 'string', nullable: true },
+  income_restrictions: { type: 'string', nullable: true },
+  filing_status: {
+    type: 'string',
+    enum: Object.values(FilingStatus),
+    nullable: true,
+  },
+  owner_status: {
+    type: 'array',
+    items: { type: 'string', enum: Object.values(OwnerStatus) },
+  },
+  other_restrictions: { type: 'string', nullable: true },
+  stacking_details: { type: 'string', nullable: true },
+  financing_details: { type: 'string', nullable: true },
+  editorial_notes: { type: 'string', nullable: true },
+  questions: { type: 'string', nullable: true },
+  serve_in_api: { type: 'boolean', nullable: true },
+} as const;
+
+// DerivedFields and its schema are associated with data not directly collected
+// in the main spreadsheets. They may have a close relationship with collected
+// fields (for example, program_start_raw and program_start). Generally, if
+// what's in the spreadsheet requires more than just fitting to an enum or
+// converting from string to number with minor cleanup, it should have a
+// separate collected and derived version.
+export type DerivedFields = {
   agi_max_limit?: number;
   agi_min_limit?: number;
-  authority_type: AuthorityType;
   authority: string;
   type: PaymentMethod; // Deprecated; we are switching to use payment_methods instead
```

Comment:
This reminds me: we can actually remove this field now (from v1); the frontend is using `payment_methods`. Fine to do that in a separate PR though.